tests/test_class_basics.rs

```diff
@@ -259,6 +260,7 @@ fn test_unsendable<T: PyClass + 'static>() -> PyResult<()> {
 
 /// If a class is marked as `unsendable`, it panics when accessed by another thread.
 #[test]
+#[cfg_attr(target_arch = "wasm32", ignore)]
```

Comment:
It might be nice to document this case as we know it's not expected to work.

```suggestion
#[cfg_attr(target_arch = "wasm32", ignore = "panic unwinding is not implemented on wasm")]
```